amr-wind/wind_energy/actuator/Actuator.cpp

```diff
@@ -63,6 +64,7 @@ void Actuator::pre_init_actions()
         pp.query("type", type);
         pp1.query("type", type);
         AMREX_ALWAYS_ASSERT(!type.empty());
+        cnt_turbfastline += (type == "TurbineFastLine") ? 1 : 0;
```

Comment:
I think it's the sum total of all OpenFAST (both `TurbFastLine` and `TurbFastDisk`) instances which must be equal to or less than the number of ranks.

Reply:
good point. Fixed.